### ISSUE TITLE
Windows 7: Failed to execute "git ls-remote --tags --heads git://github.com/Modernizr/Modernizr.git", exit code of #128

### ISSUE THREAD

**Issue by joseluisq** [yeoman/generator-webapp_DEPRECATED#314]
Hi, I have this error on Windows 7 x64 
when I generate a proyect on prompt
`yo webapp`

```
bower cached        git://github.com/Modernizr/Modernizr.git#2.6.3
bower validate      2.6.3 against git://github.com/Modernizr/Modernizr.git#~2.6.2
bower cached        git://github.com/jquery/jquery.git#1.11.0
bower validate      1.11.0 against git://github.com/jquery/jquery.git#~1.11.0
npm http 200 https://registry.npmjs.org/grunt-svgmin
bower ECMDERR       Failed to execute "git ls-remote --tags --heads git://github.com/Modernizr/Modernizr.git", exit code of #128

Additional error details:
fatal: unable to connect to github.com:
github.com[0: 192.30.252.129]: errno=No error

events.js:74
        throw TypeError('Uncaught, unspecified "error" event.');
              ^
TypeError: Uncaught, unspecified "error" event.
    at TypeError (<anonymous>)
    at Appgenerator.EventEmitter.emit (events.js:74:15)
    at done (C:\Users\Naranja\AppData\Roaming\npm\node_modules\generator-webapp\node_modules\yeoman-
generator\lib\base.js:318:16)
    at C:\Users\Naranja\AppData\Roaming\npm\node_modules\generator-webapp\node_modules\yeoman-genera
tor\node_modules\async\lib\async.js:232:13
    at C:\Users\Naranja\AppData\Roaming\npm\node_modules\generator-webapp\node_modules\yeoman-genera
tor\node_modules\async\lib\async.js:113:21
    at C:\Users\Naranja\AppData\Roaming\npm\node_modules\generator-webapp\node_modules\yeoman-genera
tor\node_modules\async\lib\async.js:24:16
    at C:\Users\Naranja\AppData\Roaming\npm\node_modules\generator-webapp\node_modules\yeoman-genera
tor\node_modules\async\lib\async.js:229:17
    at C:\Users\Naranja\AppData\Roaming\npm\node_modules\generator-webapp\node_modules\yeoman-genera
tor\node_modules\async\lib\async.js:516:34
    at Appgenerator.<anonymous> (C:\Users\Naranja\AppData\Roaming\npm\node_modules\generator-webapp\
node_modules\yeoman-generator\lib\actions\install.js:43:7)
    at ChildProcess.EventEmitter.emit (events.js:117:20)
```

**Comment by eddiemonge** [comment]
Do you have git installed? In your prompt/terminal can you run?:

``` bash
git ls-remote --tags --heads git://github.com/Modernizr/Modernizr.git
```

This should also be opened in https://github.com/bower/bower/issues

**Comment by joseluisq** [comment]
The solution is update bower to latest version
Update available: 1.3.1 (current: 1.2.8)

`npm update -g bower`
